protocol-definitions/Jet.yaml

```diff
@@ -156,6 +156,11 @@ methods:
           nullable: false
           since: 2.0
           doc: ''
+        - name: lightJobCoordinator
+          type: UUID
+          nullable: true
+          since: 5.1
+          doc: ''
```

Comment:
Optional: If you want, you can provide documentation here for the future readers